.eslintrc

```diff
@@ -5,6 +5,19 @@
         "node": true,
         "jest": true
     },
+    "rules": {
+        "no-restricted-syntax": [
+            "error",
+            {
+                "selector": "ImportDeclaration[source.value='react'] :matches(ImportNamespaceSpecifier, ImportSpecifier)",
+                "message": "Please use import React from 'react' instead."
+            },
+            {
+                "selector": "JSXFragment>JSXOpeningFragment:not(:has(JSXIdentifier))",
+                "message": "Please use React.Fragment instead."
```

Comment:
There is a dedicated [rule](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-fragments.md) in eslint-plugin-react